utilities/src/main/java/io/syndesis/qe/bdd/utils/GoogleCalendarSteps.java

```diff
@@ -0,0 +1,109 @@
+package io.syndesis.qe.bdd.utils;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventAttendee;
+import com.google.api.services.calendar.model.EventDateTime;
+import cucumber.api.DataTable;
+import cucumber.api.java.en.And;
+import io.syndesis.qe.utils.GoogleCalendarUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+public class GoogleCalendarSteps {
+
+    @Autowired
+    private GoogleCalendarUtils gcu;
+
+    @And("^created calendars$")
+    public void createdACalendarForUser(DataTable calendarsData) throws IOException {
+        List<Map<String,String>> valueRows = calendarsData.asMaps(String.class, String.class);
+        for (Map<String,String> row : valueRows) {
+            String testAccount = row.get("google_account");
+            Calendar c = new Calendar();
+            c.setSummary(row.get("calendar_summary"));
+            c.setDescription(row.get("calendar_description"));
+            gcu.insertCalendar(testAccount, c);
+        }
+    }
+
+    @And("^following \"([^\"]*)\" events in calendar \"([^\"]*)\" with account \"([^\"]*)\"$")
```

Comment:
Not clear from the step definition what it does - my first instinct was that it checks the events are in the calendar, but in fact it creates these events in the calendar